extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -478,9 +492,11 @@ protected void configure() {
                         if (!keycloakX) {
                             withFileSystemBind(realmPath.get(), KEYCLOAK_DOCKER_REALM_PATH, BindMode.READ_ONLY);
                         }
-                        realmRep = readRealmFile(filePath.toUri(), realmPath.get());
+                        realmRep = readRealmFile(filePath.toUri(), realmPath.get(), errors);
                     } else {
-                        LOG.debugf("Realm %s resource is not available", realmPath.get());
+                        errors.add(String.format("Realm %s resource is not available", realmPath.get()));
```

Comment:
I put the full error message here but it's actually not used, it's just a marker.